### ISSUE TITLE
Missing proposals and wrong .caffemodel?

### ISSUE THREAD

**Issue by Juggernaut93** [feichtenhofer/Detect-Track#31]
Hi, I was trying to test this model on ImageNet VID (no modification to the code) and I used the trained models linked in the repo homepage (like [this one](http://ftp.tugraz.at/pub/feichtenhofer/detect-track/models/ResNeXt101-DT-ilsvrc-vid.caffemodel) or [this one](http://ftp.tugraz.at/pub/feichtenhofer/detect-track/models/ResNet-101-DT-ilsvrc-vid.caffemodel)). The problem is that using those models I get completely random predictions: for each one of the 30 classes I get the same ~0.03 score, so the model doesn't detect any proposal and seems to act randomly, as it's been randomly initialized (I quadruple-checked that Caffe gets as input the right .caffemodel file).

For this reason I tried to train the model, but I constantly get missing proposal file errors. I checked and it seems that - for example - for the DET train dataset there are only ~53k proposal files while the ImageNet DET train dataset has got ~456k images. What am I missing here?

**Comment by wangpichao** [comment]
where/which folder did you put the proposals files ?

**Comment by timmeinhardt** [comment]
I experienced the same with missing proposals. The VID validation set has 176126 files and you provide 176126 proposals. However, the training set has 1122397 files but you only provide 1084113 proposals.

My proposals are in the correct folder cause it finds the once that are there but the for the difference it results in errors during training like the following:
`error: cant find file data/ILSVRC/Data/VID/proposals/RPN_proposals/ILSVRC2015_VID_train_0001/ILSVRC2015_train_00212000/002896.mat
`

I downloaded the ImageNet VID_train FTP server proposals.
